frontend/src/components/dashboard/LandingPage.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import BackgroundImage from './images/landing-page.JPG';
+import {Typography} from '@material-ui/core';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import ArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import {Link as ScrollLink} from 'react-scroll';
+
+const useStyles = makeStyles(() => ({
+  banner: {
+    background: 'url('+BackgroundImage+')',
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+    height: '100vh',
+    margin: '0',
+    position: 'relative',
+    width: '100%',
+  },
+  bannerText: {
+    left: '30%',
+    position: 'absolute',
+    top: '40%',
+    transform: 'translate(-50%,-50%)',
+    width: '100%',
+    textAlign: 'center',
+  },
+}));
+
+export default function Banner() {
+  const classes = useStyles();
+
+  return (
+    <Grid
+      container
+      direction="column"
+      justify="flex-start"
+      alignItems="center"
+      spacing={3}
+      className={classes.banner}
+    >
+      <Grid item xs={10} md={6} lg={4} className={classes.bannerText}>
+        <Typography
+          variant="h1"
+          color="primary"
+        >
+          Luminart
+        </Typography>
+        <Typography
+          variant="h4"
+          component="h2"
+          color="textPrimary"
+          gutterBottom
+        >
+          Shining a light on a new way to experience art for everyone.
```

Comment:
I like the idea of this but feel like it is a little wordy.  I am having a hard time wording a blurb myself.  I was thinking simplifying it to be "Shining a light on a new way to experience art" or even "Shining a new light on art" but I don't think those get across that the point that our website is universally accessible.  So maybe something like "Shining a light for all to enjoy art"? Still not perfect but just wanted to share a few thoughts, feel free to edit any of those, and we can brainstorm together later if you would like